### ISSUE TITLE
[WCAG 2.1]: Loading Indicators need to announce themselves to screen readers without taking focus

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#491]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
WCAG 2.1 [success criteria 4.1.3: Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html) states that things like loading indicators need to announce themselves to assistive technology without receiving keyboard focus.

## What I expected to happen
Screen readers will need to announce our Loading Indicator component without us having to manage focus. This is possible if two conditions are met:

1. The Loading Indicator `role="progressbar"` is changed to `role="status"` on [line 44](https://github.com/department-of-veterans-affairs/component-library/blob/f3bd92a053f59682a18848389285b399cb2a4f2d/src/components/LoadingIndicator/LoadingIndicator.jsx#L44)
2. The spinner *always* has a visual text label with it

## Acceptance Criteria
- [ ] No breaks in the current usability on vets-website
- [ ] Unit tests updated accordingly
- [ ] 0 axe-core violations in the unit tests
